tests/py/dynamo/partitioning/test_global_partitioning.py

```diff
@@ -1,14 +1,52 @@
 from copy import deepcopy
 
 import numpy as np
+import pytest
 import torch
+import torch.nn.functional as F
+import torch_tensorrt
 from torch.testing._internal.common_utils import TestCase, run_tests
 from torch_tensorrt.dynamo import partitioning
 
 from ..testing_utilities import lower_graph_testing
 
 
 class TestGlobalPartitioning(TestCase):
+    def test_end2end_global_partition(self):
+        class SimpleCNN(torch.nn.Module):
+            def __init__(self):
+                super(SimpleCNN, self).__init__()
+                self.conv1 = torch.nn.Conv2d(3, 16, kernel_size=3, stride=1, padding=1)
+                self.conv2 = torch.nn.Conv2d(16, 32, kernel_size=3, stride=1, padding=1)
+                self.fc1 = torch.nn.Linear(32 * 134 * 134, 10)
+
+            def forward(self, x):
+                x = F.relu(self.conv1(x))
+                x = F.max_pool2d(x, kernel_size=2, stride=2)
+                x = F.relu(self.conv2(x))
+                x = F.max_pool2d(x, kernel_size=2, stride=2)
+                x = x.view(x.size(0), -1)
+                x = self.fc1(x)
+                return x
+
+        mod = SimpleCNN().to(dtype=torch.float16, device=torch.device("cuda"))
+        mod.eval()
+        batch_size, tile_size = 1, 538
+        with torch.no_grad():
+            inputs = torch.randn(
+                batch_size, 3, tile_size, tile_size, device="cuda", dtype=torch.float16
+            )
+            try:
+                torch_tensorrt.compile(
+                    mod,
+                    ir="dynamo",
+                    inputs=[inputs],
+                    enabled_precisions={torch.float16},
```

Comment:
You should probably force fallback an op or two to make sure there are at least a few subgraphs

Reply:
added, please review again.